linux/mcompile/keymap/mc_import_rules.h

```diff
@@ -0,0 +1,38 @@
+
+#pragma once
+#ifndef MC_IMPORT_RULES_H
+#define MC_IMPORT_RULES_H
+
+class DeadKey {
```

Comment:
That`s right. 
Nevertheless I would rather not change this because I´d like to keep things the same for mcompile-windows, mcompile-linux, mcompile-mac as much as possible. (of course only if things work fine!)
If we change things we should do this on all platforms simultaneously in order for mcompile to be more maintainable.
Therefore I moved this to Issue #11802